keps/NNNN-kep-template/README.md

```diff
@@ -0,0 +1,370 @@
+<!--
+**Note:** When your KEP is complete, all of these comment blocks should be removed.
+
+To get started with this template:
+
+1. **Pick a hosting SIG.**
+  Make sure that the problem space is something the SIG is interested in taking
+  up.  KEPs should not be checked in without a sponsoring SIG.
+2. **Create an issue in kubernetes/enhancements**
+  When filing an enhancement tracking issue, please ensure to complete all
+  fields in that template.  One of the fields asks for a link to the KEP.  You
+  can leave that blank until this KEP is filed, and then go back to the
+  enhancement and add the link.
+3. **Make a copy of this template directory.**
+  Copy this template into the owning SIG's directory and name it
+  `NNNN-short-descriptive-title`, where `NNNN` is the issue number assigned to
+  your enhancement above.
+4. **Fill out as much of the kep.yaml file as you can.**
+  At minimum, you should fill in the "title", "authors", "owning-sig",
+  "status", and date-related fields.
+5. **Fill out this file as best you can.**
+  At minimum, you should fill in the "Summary", and "Motivation" sections.
+  These should be easy if you've preflighted the idea of the KEP with the
+  appropriate SIG(s).
+6. **Create a PR for this KEP.**
+  Assign it to people in the SIG that are sponsoring this process.
+7. **Merge early and iterate.**
+  Avoid getting hung up on specific details and instead aim to get the goals of
+  the KEP clarified and merged quickly.  The best way to do this is to just
+  start with the high-level sections and fill out details incrementally in
+  subsequent PRs.
+
+Just because a KEP is merged does not mean it is complete or approved.  Any KEP
+marked as a `provisional` is a working document and subject to change.  You can
+denote sections that are under active debate as follows:
+
+```
+<<[UNRESOLVED optional short context or usernames ]>>
+Stuff that is being argued.
+<<[/UNRESOLVED]>>
+```
+
+When editing KEPS, aim for tightly scoped, single-topic PRs to keep discussions
+focused.  If you disagree with what is already in a document, open a new PR
+with suggested changes.
+
+One KEP corresponds to one "feature" or "enhancement", for its whole lifecycle.
+You do not need a new KEP to move from beta to GA, for example.  If there are
+new details that belong in the KEP, edit the KEP.  Once a feature has become
+"implemented", major changes should get new KEPs.
+
+The canonical place for the latest set of instructions (and the likely source
+of this file) is [here](/keps/NNNN-kep-template/README.md).
+
+**Note:** Any PRs to move a KEP to `implementable` or significant changes once
+it is marked `implementable` must be approved by each of the KEP approvers.
```

Comment:
I think all - that's what approvers is for.

Reply:
Yeah, I think in this context (as opposed to OWNERs files), we want all of the approvers to ack.

Reply:
What I'm trying to tease out here is do we have a minimum amount of required approvers?
What if someone puts five approvers on the KEP? I think "at least two approvers are required on a KEP" or some such language makes sense.

Reply:
Is the multi-party ack enforced by tooling?  For KEPs where we have had multiple approvers, it often gets delegated or primarily assigned to a single approver in practice for final approve after multiple reviewers or sig discussions.  In general, I would expect that approvers seek quorum from other maintainers, but I think at least one is fine in practice if there is no noted disagreement after some period of time.  In general, we make folks maintainers because as a community we trust each other to maintain and are comfortable collaborating to find an outcome.

Reply:
Shouldn't we remove the other from `approvers` then?